act-rules-format.bs

```diff
@@ -51,9 +51,9 @@ In accessibility, there are often different technical solutions to make the same
   <p>If any one of these rules passes, the outcome of the composite rule for the table cell is "passed".</p>
 </div>
 
-The separation between atomic rules and composite rules can be seen as a division of responsibility. Atomic rules typically test if web content is correctly implemented in a particular solution. Composite rules can test if a combination of outcomes from atomic rules satisfy the accessibility requirement, in part or as a whole.
+The separation between atomic rules and composite rules can be seen as a division of responsibility. Atomic rules typically test if web content is correctly implemented in a particular solution. Composite rules can test if a combination of outcomes from atomic rules satisfy the accessibility requirement, in part or as a whole. Composite rules can also combine the outcome of multiple composite rules to determine a single outcome.
```

Comment:
As I read this part: 

> "Composite rules can test if a combination of outcomes from atomic rules satisfy the accessibility requirement, in part or as a whole. Composite rules can also combine the outcome of multiple composite rules to determine a single outcome." 

... composite rules can EITHER combine atomic rules OR combine composite rules. 
I don't see why a mix shouldn't be allowed, e.g.:

> "Composite rules can test if a combination of outcomes from other rules satisfy the accessibility requirement."